package.json

```diff
@@ -25,7 +25,7 @@
     "clean:examples": "rimraf examples/static",
     "less": "lessc -x src/less/styles.less ./lib/css/react-big-calendar.css",
     "assets": "cpy src/less/* lib/less",
-    "build": "npm run clean && babel src --out-dir lib && npm run assets & npm run less",
+    "build": "npm run clean && babel src --out-dir lib && npm run assets && npm run less",
```

Comment:
single `&` is also for parallel tasks on OSX/unix, but I don't think it's a good idea to run these tasks in parallel because if you wait for an exit code from the command, you'll get it before it completes, in addition to have loose output consistency (you don't know when exactly it finishes)...